xdsl/__init__.py

```diff
@@ -0,0 +1,12 @@
+from .ir import Operation
+from .builder import Builder
+
+# set up the operation callback for implicit construction
+
+
+def _op_init_callback(op: Operation):
```

Comment:
Do you think it would be possible to only set this when we are using the `implicit_region`?
In the sense that the first `implicit_region` sets this callback, and then removes it?

Reply:
It now adds the functionality when importing `xdsl.builder`